code/datums/diseases/advance/symptoms/sneeze.dm

```diff
@@ -46,12 +46,15 @@
 			if(!suppress_warning)
 				M.emote("sniff")
 		else
-			M.emote("sneeze")
```

Comment:
I'm gonna have to agree with Athath and Tim, you're removing people's way of telling they're sick until the symptoms are too late to work with.

Reply:
Sorry I'm only now getting to this. Busy couple of weeks. I'll go with the emote layout as suggested by MB.